packages/client/src/Client.js

```diff
@@ -18,47 +18,66 @@ import React from 'react';
 
 import Block from './block/Block.js';
 import Context from './Context.js';
+import DisplayMessage from './DisplayMessage.js';
 import Head from './Head.js';
 import ProgressBarController from './ProgressBarController.js';
 
 import initLowdefyContext from './initLowdefyContext.js';
 
-const Client = ({ Components, config, router, stage, types, window }) => {
-  const lowdefy = initLowdefyContext({ Components, config, router, types, stage, window });
-
+const Client = ({
+  Components,
+  config,
+  resetContext = { reset: false, setReset: () => undefined },
+  router,
+  stage,
```

Comment:
Do we still use `stage` with resetContext?

Reply:
Yes, to add lowdefy to window.